src/Ocean/SplitExplicit01/OceanBoundaryConditions.jl

```diff
@@ -131,9 +131,9 @@ apply no penetration boundary for temperature
     A⁻,
     t,
 )
-#   D⁺.ν∇u = -D⁻.ν∇u
```

Comment:
I guess we really shouldn't have commented out lines in commits. Anyways, if we do note for the future that's better to put the comment in the place of code, ie, we shouldn't add comments at the very beginning of the line (like you'd do with fortran) since we'll end up with double indents after formatting.